### DISCUSSION TITLE
v1.0.7版本的源码可以用MSVC2015编译吗

### DISCUSSION THREAD

**Discussion by TreatTrick** [CefView/QCefView#283] [Q&A] [answered]
您好， 非常感谢你们提供的库，像现在想把这个库用到一个用MSVC 2015编译的Qt小项目里面，但是在我的电脑上用MSVC 2015编译QCefView失败了。我不知道是我电脑的环境问题还是必须要用高版本编译器

**Comment by tishion** [comment, marked as answer]
我没记错的话，2015应该不支持C++17，如果非要用，你需要自己适配一下。
